### PR TITLE
www.qvcuk.com

### PR DESCRIPTION
**Pull request by SKEIDs**
https://github.com/AdguardTeam/AdguardFilters/issues/102434

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+qvcuk.com##div.footer-privacy-banner
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:115]: Please add this domain to the existing rule. ( `qvc.de##.footer-privacy-banner` )

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/cookies_specific.txt:115]: Thank you

Comment (Yuki2718) [reply AnnoyancesFilter/sections/cookies_specific.txt:115]: @Sergey-Lyapin requested you to remove `qvcuk.com##div.footer-privacy-banner` line and instead add domain to the existing rule `qvc.de##.footer-privacy-banner`, the same as what I requested in https://github.com/AdguardTeam/AdguardFilters/pull/102372 and https://github.com/AdguardTeam/AdguardFilters/pull/102373

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Hm, I didn't notice your PR and already fixed this issue, sorry. 
Please take a look how it should be:
https://github.com/AdguardTeam/AdguardFilters/commit/d1e72522aefdb49ded7dafaa3fd4f7208c1c3fcc

**Pull request closed without merging**